src/SDK/Language/DotNet.php

```diff
@@ -181,26 +182,20 @@ public function getParamDefault(array $param)
             switch ($type) {
                 case self::TYPE_NUMBER:
                 case self::TYPE_INTEGER:
+                case self::TYPE_ARRAY:
```

Comment:
why not use something similar to [] for array?

Reply:
Because with the data type `List<object>` and a value of `null`, it becomes an empty array. That was confusing to me as well 👍